packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -654,22 +697,42 @@ class _GenericCard extends Component<GenericCardPropsInternal> {
               </Box>
             )}
           </div>
-          {includeCitation && (
-            <Box
-              sx={{
-                marginLeft: 'auto',
-                paddingTop: '21px',
-                paddingRight: '40px',
-              }}
-            >
-              <CitationPopover
-                title={title}
-                doi={doiValue}
-                boilerplateText={citationBoilerplateText}
-                defaultCitationFormat={defaultCitationFormat}
-              />
-            </Box>
-          )}
+
+          {includeCitation ||
+            (downloadCartSynIdValue && (
```

Comment:
These parentheses are wrong, it should be

```suggestion
          {(includeCitation || downloadCartSynIdValue) && (
```